POS/Integration/06a_CreateStoreManually.md

```diff
@@ -0,0 +1,500 @@
+- The global settings for POS are configured, see [Configure the global settings for POS](./05_ConfigureGlobalSettings.md).
+
+
+#### Procedure
+
+*Venduo POS > Management > Tab STORES*
+
+![Create Store](../../Assets/Screenshots/POS/Management/Stores/Stores.png "[Create Store]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    The [Create store via wizard (recommended) ![Store Wizard](../../Assets/Icons/Tool.png "[Store Wizard]") ] button and the [Create Store ![Store Manual](../../Assets/Icons/Plus02.png "[Store Manual]") ] button are displayed.
+
+2. Click the [Create Store ![Store Manual](../../Assets/Icons/Plus02.png "[Store Manual]")] button to add a new store manually.   
+    The *Create Store* window is displayed.
+
+    ![Create Store](../../Assets/Screenshots/POS/Management/Stores/CreateStore.png "[Create Store]")
+
+3. Enter a store name in the *Name* field.
+
+4. Click the [CREATE] button.   
+    The *Creating Store...* message is displayed. It may take several minutes until the creation is completed. The new store is displayed in the *STORES* tab. The *Basic data* tab is displayed by default.
+
+    ![Store Details](../../Assets/Screenshots/POS/Management/Stores/Store/BasicData/BasicData.png "[Store Details]")
+
+    The following entities have been automatically created:
+    - *DataHub* module:   
+        The following attribute sets have been created<!--- Ist das richtig?-->:
+        - Orders channel POS Store "Name of new store"
+        - Line items channel POS Store "Name of new store"
+        - Payment items channel POS Store "Name of new store"
+        - Discount items channel POS Store "Name of new store" 
+        - Returns connection POS Store "Name of the new store"
+        - Return line items channel POS Store "Name of the new store"
+        - Redeem voucher item channel POS Store "Name of the new store"
+        - Service items channel POS Store "Name of the new store"  
+        - Set for POS store "Name of new store"
+        - Offer option set for POS store "Name of the new store" 
+    - *PIM* module   
+        Your customer-specific PIM attribute sets have been created for the new store with a destination attribute set *Set for POS store "Name of the new store"*
+    - *Omni-Channel* module   
+    The following attribute sets have been created:    
+        - A connection *POS store "Name of new store"*
+        - A variant set *Set for POS for "Name of the new store"* 
+        - A variant set *Offer option set for POS store "Name of the new store"* 
+
+
+## Configure the store details
+
+In the store details, you give some detailed information about your store. Further, you assign a store manager.
+
+#### Prerequisites
+
+- A store has been created, see [Create a store manually](#create-a-store-manually) or [Create a store using the wizard](./06_CreateStore.md#create-a-store-using-the-wizard).
```

Comment:
or [Create a store using the wizard](./06_CreateStore.md#create-a-store-using-the-wizard). --> Trifft dann das hier zu? Muss man nicht Wizard zu Ende führen? Wenn man Wizard vor dem Finalize abbricht, ist der Store trotzdem erstellt?

Reply:
ja